OpenRA.Game/Graphics/Video.cs

```diff
@@ -13,13 +13,16 @@ namespace OpenRA.Video
 {
 	public interface IVideo
 	{
-		ushort Frames { get; }
+		ushort FrameCount { get; }
 		byte Framerate { get; }
 		ushort Width { get; }
 		ushort Height { get; }
-		uint[,] FrameData { get; }
 
-		int CurrentFrame { get; }
+		/// <summary>
+		/// Current frame color data in 32-bit BGRA.
+		/// </summary>
+		byte[] CurrentFrameData { get; }
+		int CurrentFrameNumber { get; }
```

Comment:
Overthinking/nitpicking: `CurrentFrameIndex`? `Index` strongly suggests 0-indexing to me, whereas `Number` feels more 1-indexey.